Cython/Compiler/PyrexTypes.py

```diff
@@ -1314,12 +1314,16 @@ def check_for_null_code(self, cname):
 
     def generate_incref(self, code, cname, nanny):
         if nanny:
+            if code.funcstate:
```

Comment:
In which case do we not have a funcstate? Maybe we should get rid of that case instead?
I can't think of a place where we generated increfs outside of user/wrapper functions or the module init function.